src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptor.cs

```diff
@@ -28,6 +46,7 @@ public class TagHelperDescriptor
         /// Instantiates a new instance of the <see cref="TagHelperDescriptor"/> class with the given
         /// <paramref name="attributes"/>.
         /// </summary>
+        /// <param name="prefix">The prefix of the HTML element name the tag helper targets.</param>
```

Comment:
the information to put in the extremely dumb `TagHelperDescriptor` is what the code requires.  everything is doing matches against either the short or the long name.  so provide those.  don't force repeated concatenations or surface a "prefix" concept when the code just wants a single string to match.